src/AutoRest.CSharp/Common/Generation/Types/CSharpType.cs

```diff
@@ -163,18 +163,6 @@ internal static CSharpType FromSystemType(Type type, string defaultNamespace, So
         internal static CSharpType FromSystemType(BuildContext context, Type type)
             => FromSystemType(type, context.DefaultNamespace, context.SourceInputModel);
 
-        public bool IsCollectionType()
```

Comment:
This method is removed because this method is never used and we have the same method in `TypeFactory` which is widely used everywhere